### ISSUE TITLE
I get the crash report after latest windows update

### ISSUE THREAD

**Issue by vladegikyan** [docker/for-win#8823]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: A29AC363-6075-43A6-81FC-8F3AF7BE9771/20201005043712

### Information

  - Windows Version: Windows 10
  - Docker Desktop Version: 2.4.0.0. Community edition
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: No

### Steps to reproduce the behavior

  1. Start the Docker Desktop
  2. Docker failed to initialize with
System.NullReferenceException:
Object reference not set to an instance of an object.
   at Docker.Core.Cli.CliOptionsLoader.Get() in C:\workspaces\stable-2.4.x\src\github.com\docker\pinata\win\src\Docker.Core\cli\CliOptionsLoader.cs:line 33
   at Docker.Program.Run(IReadOnlyCollection`1 args) in C:\workspaces\stable-2.4.x\src\github.com\docker\pinata\win\src\Docker.Desktop\Program.cs:line 327

**Comment by simonferquel** [comment]
@vladegikyan can you send me your ~/.docker/config.json ? this is an issue I am actively working on, and I'd like to see a sample of CLI config that triggers this.

**Comment by vladegikyan** [comment]
Hello, Simon.
It is empty.

**Comment by simonferquel** [comment]
Thanks @vladegikyan (empty meaning an empty text file, or an empty json object such as `{}` ?).
You can probably just remove the file then, Docker Desktop will create a default one with default settings for you.

**Comment by LightSnowDev** [comment]
I got exactly the same issue, but with a corrupted file.
The corrupted file came from a (non related) bluescreen, leaving only unreadable binary stuff.

Deleting the .docker folder solved the issue.

But:
Shouldn't the "delete/purge" Button resolve such an issue?
Or the "reset configuration" Button, which can be clicked inside the "bug"-window?
Or when reinstalling docker?

I tried all three options, apparently non of the deleted the config file.

[Running Docker-Desktop 2.4.0.0. on Windows (Bugsnag report e7956e0f-78c5-44f4-b32a-6cdffae5ab6a)]
